odpc.client/src/features/waardelijst/components/WaardelijstenForm.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <fieldset>
+    <legend>Waardelijsten</legend>
+
+    <checkbox-group
+      v-if="organisaties.length"
+      :title="WAARDELIJSTEN.ORGANISATIE"
+      :options="organisaties"
+      v-model="model"
+    />
+
+    <checkbox-group
+      v-if="informatiecategorieen.length"
+      :title="WAARDELIJSTEN.INFORMATIECATEGORIE"
+      :options="informatiecategorieen"
+      v-model="model"
+    />
+  </fieldset>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import CheckboxGroup from "@/components/checkbox-group/CheckboxGroup.vue";
+import { WAARDELIJSTEN, organisaties, informatiecategorieen } from "../";
```

Comment:
iets te veel magic naar mijn smaak: waar komen die informatiecategorien vandaan, is geen hardcoded lijstje, ergens zal je moeten wachten tot ze geladen zijn, dat gebeurd weer ergens anders. beetje een zoekpuzzel, waarom niet op een plek laden, loading/errorstatus ed afhandelen en het resultaat verder meegeven als prop. maakt  dat de code niet makkelijker te volgen?

Reply:
Ik denk inderdaad dat het onoverzichtelijk is dat de daadwerkelijke informatiecategorieën  en de loading state die erbij hoort op verschillende plekken afgehandeld worden. Ik weet niet of ik prop drilling een beter alternatief vind. Misschien dat Suspense hier een oplossing kan bieden.

Reply:
suspense is helaas nog steeds een experimentele feature. liever niet gebruiken

Reply:
8d322ea354fe8016e9d1ab6f955c5718a6b67788